ext/standard/tests/general_functions/sleep_basic.phpt

```diff
@@ -31,9 +41,31 @@ if ($time >= $sleeplow) {
 } else {
     echo "TEST FAILED - time is {$time} secs and sleep was {$sleeptime} secs\n";
 }
+if (!have_usleep()) {
+    // ¯\_(ツ)_/¯
+    echo "FRACTIONAL SLEEP TEST PASSED\n";
+    echo "Fractional sleep return value: 0\n";
```

Comment:
```suggestion
    echo "Fractional sleep return value: 0\n";
    echo "FRACTIONAL SLEEP TEST PASSED\n";
```

not sure if the test should really pass if not supported, but the lines are definitely swapped than what is expected